Neptune.Models/DataTransferObjects/OnlandVisualTrashAssessmentAreaDetailDto.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Neptune.Models.DataTransferObjects;
+﻿using NetTopologySuite.Geometries;

```

Comment:
clean up usings? I don't think you need this reference since you don't have a Geometry object